docs/user/cassandra_datastore.rst

```diff
@@ -46,16 +46,11 @@ installation directory).
 
 After you've successfully installed and configured Cassandra, you can
 move on to installing the GeoMesa Cassandra distribution.
-Start by choosing or creating a directory where you want
-to put it. Then ``cd`` into that directory and type::
-
-    wget https://repo.locationtech.org/content/repositories/geomesa-releases/org/locationtech/geomesa/geomesa-cassandra-dist_2.11/$VERSION/geomesa-cassandra-dist_2.11-$VERSION-bin.tar.gz
```

Comment:
it seems like this repo is still valid - there are m1 and m2 versions there now. The 'snapshots' repo has dists from as recently as today.